ds/data/processed/eurostat/eurostat_private_households_income.nuts2.csv

```diff
@@ -0,0 +1,267 @@
+year,nuts_id,nuts_year_spec,eurostat_private_households_income
+2012,UKH2,2013,23500.0
```

Comment:
shouldn't `nuts_year_spec` here be `2010`?

Reply:
I think this may be one of the indicators where the data is only available at particular NUTS versions? @dede95 can you clarify?

Reply:
@mindrones Yes, what George said! The data for this indicator is only available at the 2013 version NUTS. I think this is the case for a few other Eurostat indicators too.

Reply:
ah I see, OK then!